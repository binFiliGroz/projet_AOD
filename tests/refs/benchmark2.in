 8  3  9  5  1  6  7  2  1  9 