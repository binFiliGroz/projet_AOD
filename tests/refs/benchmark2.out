static long BSTdepth = 131; // pour info. Non demand√©
static int BSTroot = 2;
static int BSTtree[10][2] = {
{-1, 1}, 
{-1, -1}, 
{0, 6}, 
{-1, 4}, 
{-1, -1}, 
{3, -1}, 
{5, 9}, 
{-1, 8}, 
{-1, -1}, 
{7, -1} };
static int opt =131;
static int p[10] = {
8, 
3, 
9, 
5, 
1, 
6, 
7, 
2, 
1, 
9};
